README.md

```diff
@@ -57,7 +57,10 @@ To run the OAuth example, you will need to have your Kafka cluster configured wi
 ## Configuration
 
 Although this Hello World is simple example it is fully configurable.
-Below are listed and described environmental variables.
+Below are listed and described environmental variables. 
```

Comment:
I think we use "environment variable" everywhere in our docs. Can we change this way everywhere you are using "environmental"?

Reply:
Done